beaver/transports/sqs_transport.py

```diff
@@ -81,15 +86,18 @@ def callback(self, filename, lines, **kwargs):
         return True
 
     def _send_message_batch(self, message_batch):
-        try:
-            result = self._queue.write_batch(message_batch)
-            if not result:
-                self._logger.error('Error occurred sending messages to SQS queue {0}. result: {1}'.format(
-                    self._queue_name, result))
-                raise TransportException('Error occurred sending message to queue {0}'.format(self._queue_name))
-        except Exception, e:
-            self._logger.exception('Exception occurred sending batch to SQS queue')
-            raise TransportException(e.message)
+        for queue in self._queue:
```

Comment:
That's as designed. An example use case being the replication of data to prod and dev logstash instances. AWS claim to support "unlimited" queued SQS messages, so I'm not sure the use case you described would provide any benefit.